Add `Skinning Contest: Icons of an Era - Results` news post and update `Skinning Contest #4` wiki page (#9868)
ppy/osu-wiki: news/2023/2023-08-05-skinning-contest-icons-of-an-era-results.md

---
layout: post
title: "Skinning Contest: Icons of an Era - Results"
date: 2023-08-05 00:00:00 +0000
---

Tune in to find out the results for the latest skinning contest, and snag yourself some amazing skins centred around the past 15 years of osu!

![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/banner.jpg)

> The community has spoken with **3792** players casting a total of **9981** votes across **16** different top-of-the-notch skins, and out of all entries, the following 3 entries have climbed their way to the podium:

Reviewer: ```suggestion
The community has spoken with **3792** players casting a total of **9981** votes across **16** different top-of-the-notch skins, and out of all entries, the following 3 have climbed their way to the podium:
```

## 3rd Place - QUAVER: DIVIDE BY 8

**1022** quaver retry spammers decided [thetasigma](https://osu.ppy.sh/users/6234482)'s [QUAVER: DIVIDE BY 8](https://osu.ppy.sh/community/forums/topics/1792739) was worthy of their votes, netting themselves the **3rd** place!

> Based on [Sotarks](https://osu.ppy.sh/users/4452992)' [quaver](https://osu.ppy.sh/beatmapsets/873811#osu/1859185) beatmap set, this skin fully embodies the set's background and incorporates the universe theme across the entire skin. The song selection screen is beautifully done with various planets and stars floating about, and even [Stardust](https://vocaloid.fandom.com/wiki/Xingchen) from the original background makes an appearance across several gameplay screens. Congratulations to thetasigma for achieving 3rd place!

Reviewer: ```suggestion
Based on [Sotarks](https://osu.ppy.sh/users/4452992)' [quaver](https://osu.ppy.sh/beatmapsets/873811#osu/1859185) beatmap, this skin fully embodies the map's background and incorporates the universe theme across the entire skin. The song selection screen is beautifully done with various planets and stars floating about, and even [Stardust](https://vocaloid.fandom.com/wiki/Xingchen) from the original background makes an appearance across several gameplay screens. Congratulations to thetasigma for achieving 3rd place!
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/dVbEeRi0y_A" frameborder="0" allowfullscreen></iframe>
</div>

![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/3rd_badge@2x.png)

## 2nd Place - Everlasting Moment

Time stood still as **1058** voters cast their votes, resulting in a very close battle between 3rd and 2nd place. In the end, [Chromasia](https://osu.ppy.sh/users/7306251) was able to win the battle and achieve **2nd** place with their entry!

> [Everlasting Moment](https://osu.ppy.sh/community/forums/topics/1792308) keeps up with the universe theme, while providing a unique flair based off of the background art used in several beatmap sets of [Release Hallucination](https://osu.ppy.sh/beatmaps/artists/194)'s Chronostasis. The colour palette is in line with the original art, and that makes this skin very pleasing to use! Congratulations to Chromasia on their 2nd place!

Reviewer: ```suggestion
[Everlasting Moment](https://osu.ppy.sh/community/forums/topics/1792308) keeps up with the universe theme, while providing a unique flair based on the background art used in several beatmaps of [Release Hallucination](https://osu.ppy.sh/beatmaps/artists/194)'s *Chronostasis*. The colour palette is in line with the original art, and that makes this skin very pleasing to use! Congratulations to Chromasia on their 2nd place!
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/MZHOHYw1ysw" frameborder="0" allowfullscreen></iframe>
</div>

![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/2nd_badge@2x.png)

## Winner - FOOL MOON NIGHT

> [Spoo](https://osu.ppy.sh/users/11805037) was not a *fool* when they chose their theme (which somehow again includes a universe/space theme), because this skin stands head and shoulders above all other entries with a mind-blowing **1303** votes!

Reviewer: ```suggestion
[Spoo](https://osu.ppy.sh/users/11805037) was not a *fool* when they chose their theme (which somehow again involves the universe or space), because this skin stands head and shoulders above all other entries with a mind-blowing **1303** votes!
```

> Inspired from the 2016 set of ["A FOOL MOON NIGHT"](https://osu.ppy.sh/beatmapsets/524026#osu/1186901) by The Koxx, [Spoo's entry](https://osu.ppy.sh/community/forums/topics/1775136?n=1) completely blew the minds of the community by hand drawing **everything** in the style of the original background. The quality of this skin is impeccable, and the **1st** place is absolutely deserved. Congratulations to Spoo for their well-deserved victory!

Reviewer: ```suggestion
Inspired from the [2016 map of *A FOOL MOON NIGHT*](https://osu.ppy.sh/beatmapsets/524026#osu/1186901) by The Koxx, [Spoo's entry](https://osu.ppy.sh/community/forums/topics/1775136?n=1) completely blew the minds of the community by hand-drawing **everything** in the style of the original background. The quality of this skin is impeccable, and the **1st** place is absolutely deserved. Congratulations to Spoo for their well-deserved victory!
```

>  <div align="center">

Reviewer: ```suggestion
<div align="center">
```

    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/tYEMQL28rqg" frameborder="0" allowfullscreen></iframe>
</div>

> ![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/1st_badge@2x.png)

Author: I originally just had another sentence at the bottom congratulating them, then I thought it was stupid to have such a small sentence at the bottom so I put it back up.

Unsure how to resolve this without additional fluff, maybe I could just do the same as the last iteration lol

> ![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/1st_badge@2x.png)

Author: 0cda37027a52a3f7fa9f214c234deabfc7df4248 should be OK I think, @RockRoller01 please also take a look

> ![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/1st_badge@2x.png)

Reviewer 2: Overall not really feeling the paragraphs added, the one for max seems fine but the others are ugh. Not to mention "number of entries to a contest" is actually a statistic that means nothing, like absolutly nothing at all..

Alternatives for first and second place paragraphs should be sought out

> ![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/1st_badge@2x.png)

Author: Yeah I don't personally feel the paragraphs are necessary, maybe we should just say "here is the shinning badge that they will be receiving on their profiles" or something like that

</div>

Considering the fact that this is the 2nd ever skin created by Chromasia for an official skinning contest, we are very impressed with the quality of their work and cannot wait to see what else they have to offer. Congratulations on the 2nd place!

![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/2nd_badge@2x.png)

## Winner - FOOL MOON NIGHT

[Spoo](https://osu.ppy.sh/users/11805037) was not a *fool* when they chose their theme (which somehow again involves the universe or space), because this skin stands head and shoulders above all other entries with a mind-blowing **1303** votes!

Inspired from the [2016 map of *A FOOL MOON NIGHT*](https://osu.ppy.sh/beatmapsets/524026#osu/1186901) by The Koxx, [Spoo's entry](https://osu.ppy.sh/community/forums/topics/1775136?n=1) completely blew the minds of the community by hand-drawing **everything** in the style of the original background. The quality of this skin is impeccable, and the **1st** place is absolutely deserved.

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/tYEMQL28rqg" frameborder="0" allowfullscreen></iframe>
</div>

> Spoo is also someone who has only had 2 entries for an official contest, however the effort they put into this skin blew everything else out of the water, as apparent by the 300 vote gap between 1st and 2nd place. Congratulations on your very well deserved first victory!

Reviewer 2: > Spoo is also someone who has only had 2 entries for an official contest

This sentences does nothing, should be cut

If you want to do a paragraph about contest entries should probably be smth like "With only their second submission into an official contest they already got the winning spot"

Reviewer 2: Just read 2nd place paragraph (somehow skipped that, oops) and that would be quite similar to it...
